restaurants/views.py

```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from django.http import JsonResponse
+from django.views import View
 
-# Create your views here.
+from restaurants.models import SubCategory
+
+class MainListView(View):
+    def get(self, request):
+        try:
+            subcategorys = SubCategory.objects.all()           
+            subcategory_list = []
+            for subcategory in subcategorys:                 
+                subcategory_list.append({
+                    "image" : subcategory.restaurants.all()[0].foods.all()[0].images.all()[0].image_url                
```

Comment:
subcategory의 image를 상품에서 가져오는 건가요??
그리고 다른 정보들은 필요 없나요? (id, name 등..)

category테이블에 image_url을 추가하는게 코드가 깔끔해질 것 같습니다!

```suggestion
                    "image" : subcategory.restaurants.all()[0].foods.all()[0].images.all()[0].image_url                
```

Reply:
네. 이미지만 가져오려고 했습니다. DB에서 image_url에 144개가 있는데, 저희가 sub_category에 맞는 첫번째에만 사진을 넣어놔서 [0]으로 써서 가져왔는데요. 그럼 annotate로 category table에 image_url field를 만들라는 말씀이신가요? annotate라는 메서드는 많이 떨어져 있는 메서드라도 아무렇게나 사용가능한 건가요?